### PR TITLE
Clarify class keyword usage

### PR DESCRIPTION
**Pull request by nicolas-scarrci**
* *Bugsquad edit, closes: godotengine/godot#79974*

### CODE REVIEW

**Review by kleonc — COMMENTED** [review]
A similar link could be added for `class_name`, pointing to the `Registering named classes` section.

File: tutorials/scripting/gdscript/gdscript_basics.rst [hunk 1]
```diff
@@ -668,7 +668,7 @@ Engine built-in types
 
 :ref:`Color <class_Color>`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
-
+Registering named classes
```
Comment (dalexeev) [inline tutorials/scripting/gdscript/gdscript_basics.rst:671]: This looks like a mistake. The header already exists.

https://github.com/godotengine/godot-docs/blob/57ef6737573158a2033b34ef9fe4bdeb14e07f80/tutorials/scripting/gdscript/gdscript_basics.rst?plain=1#L1574-L1580

File: tutorials/scripting/gdscript/gdscript_basics.rst [hunk 2]
```diff
@@ -173,7 +173,7 @@ in case you want to take a look under the hood.
 +------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
 | return     | Returns a value from a function.                                                                                                                  |
 +------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
-| class      | Defines a class.                                                                                                                                  |
+| class      | Defines an inner class. See `Classes`_.                                                                                                           |
```
[the remark this answers is not in the document]
Comment (AThousandShips) [reply tutorials/scripting/gdscript/gdscript_basics.rst:176]: This still hasn't been applied

**Comment by nicolas-scarrci** [comment]
Is there some next step I should take? Or can I safely consign it to your care and forget about this pull request entirely?

**Review by dalexeev — APPROVED** [review]

**Review by mhilbrunner — APPROVED** [review]

**Pull request merged**